src/client.rs

```diff
@@ -17,7 +17,11 @@ impl<T: DaprInterface> Client<T> {
     ///
     /// * `addr` - Address of gRPC server to connect to.
     pub async fn connect(addr: String) -> Result<Self, Error> {
```

Comment:
Can we also support `port: Option<String>` that overrides the GRPC port environment variable?
Also, if the environment variable does not exist, it can fallback to a standard default, like `50051`.